Gemfile

```diff
@@ -5,9 +5,9 @@ platforms :ruby do
 end
 
 group :test do
-  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 4.2'])
+  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 5.0'])
```

Comment:
I don't understand that broad specification (nor that RAILS_VERSION) for a test gem, can't we simply go with the latest?
If we need to ensure for some obscure reason that it works with different AR versions the way to go would be to use Appraisal.
Also, it should be moved to the gemspec file.